docs/reference/pc_channel.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Package Control: channel.json"
+---
+
+<!-- Original: https://github.com/wbond/package_control/blob/master/example-channel.json -->
+
+
+# Examples for the channel.json
+
+The latest version of the default channel.json
+can be found in the crawler release assets:  
+https://github.com/packagecontrol/thecrawl/releases/tag/the-channel
+
+
+## Properties
+
+- $schema
+- schema_version
+- repositories
+- packages_cache
+- libraries_cache
+
+"$schema" is used by LSP and LSP-json to provide linting and completions.
+"schema_version" is currently at 4.0.0.
+
+```json
+{
+    "$schema": "sublime://packagecontrol.io/schemas/channel",
+    "schema_version": "4.0.0",
+    "repositories": [],
+    "packages_cache": {},
+    "libraries_cache": {}
+}
+```
+
+
+### repositories
+
+The "repositories" array lists the repositories to be includes in the channel.
+All repositories must be an HTTPS URL, or a local or relative file path.
+
+The following values are supported:
+
+- an external repository.json URL
+- a repository located relative to this one, via a relative path
+- a `file:///` protocol URL
+- a single GitHub repository URL
+  (this is mostly useful for users adding a single package repository
+  in their Sublime installation)
+- a GitHub user or organization
+  (possible, but not recommended)
```

Comment:
```suggestion
- an external `repository.json` URL
- a repository located relative to this one, via a relative path
- a `file:///` protocol URL
- a single GitHub repository URL
  (this is mostly useful for users adding a single package repository
  in their Sublime installation)
- a GitHub user or organization
  (possible, but not recommended)
```

Also, consider not assuming GitHub (although it's overwhelmingly likely...)

Reply:
> Also, consider not assuming GitHub (although it's overwhelmingly likely...)

Its's what the docs had and I don't know for sure whether all of this specifically is fully supported otherwise.